hpx/include/parallel_for_each.hpp

```diff
@@ -4,12 +4,11 @@
 //  Distributed under the Boost Software License, Version 1.0. (See accompanying
 //  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
-#if !defined(HPX_PARALLEL_FOREACH_JUN_28_2014_0827AM)
-#define HPX_PARALLEL_FOREACH_JUN_28_2014_0827AM
+#if !defined(HPX_PARALLEL_FOREACH_JUN_21_2017_0827AM)
+#define HPX_PARALLEL_FOREACH_JUN_21_2017_0827AM
 
```

Comment:
I think it's simply a misunderstanding between me and the student. I told him to modify include guards but I meant only files he has created.

@ajaivgeorge could you revert those changes in files which have not been created by you?